scripts/prereg/approve_draft_registrations.py

```diff
@@ -5,7 +5,7 @@
 import logging
 
 from website.app import init_app
-from website.models import DraftRegistration, Sanction, User
+from website.project.model import DraftRegistration, Sanction, User
```

Comment:
This seems to have been done because Sanction is no longer imported in website.models. User however is also no longer used.